Prox/Prox/Utilities/GoogleDirectionsMatrixTravelTimesProvider.swift

```diff
@@ -43,8 +46,71 @@ class GoogleDirectionsMatrixTravelTimesProvider: TravelTimesProvider {
         return AFURLSessionManager(sessionConfiguration: configuration)
     }()
 
+    static func travelTimes(fromLocation: CLLocationCoordinate2D, toLocations: [PlaceKey : CLLocationCoordinate2D], byTransitType transitType: MKDirectionsTransportType) -> Deferred<DatabaseResult<[TravelTimes]>> {
+        NSLog("Calculating travel times for \(toLocations)")
+        let locations = Array(toLocations.values)
+        var index = 0
+        var fetchSize = min(locations.endIndex, 25) - 1
```

Comment:
I'll add a comment, but it's because the API has a limit on 25 destinations per request.